content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
 
 The decision you make will be entered during setup.
 
+#### Mounted Disk Guidelines
+
+The mounted disk option provides for significant flexibility in how PTFE stores it's data. In order to help customers narrow down possibilites we're providing some guidelines about how to use mounted disk.
+
+The follow are **supported** mounted disk types:
+* AWS EBS
+* GCP Zonal Persistent Disk
+* Azure Disk Storage
+* iSCSI
+
+The follow are **not supported** mounted disk types:
+* NFS
+* SMB/CIFS
+
+The reason for supporting only some types of mounted disk is to provide relability and performance guarantees required to store and retrieve the data PTFE utilizes.
```

Comment:
The supported mounted disk types provide the necessary reliability and performance for data storage and retrieval in PTFE.